src/main/java/org/kohsuke/github/GitUser.java

```diff
@@ -42,8 +42,6 @@ public String getEmail() {
      *
      * @return GitHub username
      */
-    @Preview
-    @Deprecated
     @CheckForNull
```

Comment:
This method was not labeled properly. The field itself is not part of a preview API. It is only optionally present depending on the source of the data. We are already denoting the lack of presence with the `@CheckForNull` annotation.